lib/sidekiq/statistic/log_parser.rb

```diff
@@ -57,6 +57,10 @@ def log_file
       def last_log_lines
         Sidekiq::Statistic.configuration.last_log_lines
       end
+
+      def log_line_contains_worker_regexp
+        /([\W]+|^)#{@worker_name}([\W]+|$)/
```

Comment:
let's move it to constant and allocate regexp one time instead using method and allocate this regexp each time